cmd/statusd/main.go

```diff
@@ -37,6 +37,8 @@ var (
 	configFiles      configFlags
 	logLevel         = flag.String("log", "", `Log level, one of: "ERROR", "WARN", "INFO", "DEBUG", and "TRACE"`)
 	logWithoutColors = flag.Bool("log-without-color", false, "Disables log colors")
+	ipcEnabled       = flag.Bool("ipc", false, "Enable IPC RPC endpoint")
```

Comment:
Can't we just have `-ipc` to enable it? What's the use case for a specific IPC path? It can be controlled with `datadir` as the ipc file is inside of it.

Reply:
I see, that's a fair point 👍

Reply:
Would it make sense to merge the two and assume that if you're passing a path, you're implicitly enabling IPC? Since we're not computing a default path anymore, I don't think there's value in passing `-ipc` and not `-ipcfile`, right?

Reply:
@PombeirP hmm, there is some value - if people assume default behaviour of `-ipc` flag to create `geth.ipc` file in well-known location (I guess, we should try to be consistent with geth users' expectations), using `-ipc` without `-ipcfile` makes sense. You may tell people "just run `./statusd ipc` and geth attach to the file printed in logs".